<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Termometro</class>
 <widget class="QMainWindow" name="Termometro">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>253</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Termometro</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Recursos.qrc">
    <normaloff>:/icono</normaloff>:/icono</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Centigrados</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLCDNumber" name="lcdCent">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 255, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="dialCent">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="minimum">
          <number>-273</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Fahrenheit</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLCDNumber" name="lcdFahr">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(0, 255, 0);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="dialFahr">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="minimum">
          <number>-460</number>
         </property>
         <property name="maximum">
          <number>210</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="title">
       <string>Kelvin</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLCDNumber" name="lcdKel">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 85, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDial" name="dialKel">
         <property name="cursor">
          <cursorShape>OpenHandCursor</cursorShape>
         </property>
         <property name="maximum">
          <number>375</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>dialCent</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdCent</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialFahr</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdFahr</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialKel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdKel</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
